<launch>
    <!-- model_name: resnet50/resnet50int8/swint -->
    <!-- precision: fp16/int8 -->
    <param name="model_name" type="string" value="swint" />
    <param name="precision" type="string" value="fp16" />

    <!-- ros topic name -->
    <param name="topic_cloud"  type="string" value="/lidar_top" />
    <param name="topic_img_f"  type="string" value="/cam_front/raw" />
    <param name="topic_img_fl" type="string" value="/cam_front_left/raw" />
    <param name="topic_img_fr" type="string" value="/cam_front_right/raw" />
    <param name="topic_img_b"  type="string" value="/cam_back/raw" />
    <param name="topic_img_bl" type="string" value="/cam_back_left/raw" />
    <param name="topic_img_br" type="string" value="/cam_back_right/raw" />

    <!-- bevfusion node -->
    <node pkg="bevfusion" name="bevfusion_node" type="bevfusion_node" output="screen" />
    
    <!-- rviz -->
    <node pkg="rviz" name ="rviz_bevfusion_node" type="rviz"
        respawn="false" output="screen" args="-d $(find bevfusion)/launch/1.rviz"/>
</launch>